text-run/command-description.js

```diff
@@ -0,0 +1,51 @@
+const child_process = require('child_process')
+const diff = require('jsdiff-console')
+const getCommand = require('./helpers/get-command.js')
+
+module.exports = async function (activity) {
+  const mdDesc = getMd(activity)
+  const cliDesc = getCliDesc(activity)
+  diff(mdDesc, cliDesc)
+}
+
+function getMd (activity) {
+  return activity.nodes
+    .map(nodeContent)
+    .map(text => text.trim())
+    .join('\n')
+    .replace(/\./g, '.\n')
+    .replace(/\,/g, ',\n')
+    .replace(/:/g, '\n')
+    .replace(/"/g, '\n')
+    .replace(/\n<\/?a>\n/g, ' ')
+    .replace(/[ ]+/g, ' ')
+    .replace(/ \./g, '.')
+    .replace(/ \,/g, ',')
+    .replace(/\n+/g, '\n')
+    .replace(/^\s+/gm, '')
+    .replace(/\s+$/gm, '')
```

Comment:
Some of these normalizations appear duplicated between `getMd` and `getCliDesc`. Maybe a general normalize function?

Reply:
good idea, done